ecs.tf

```diff
@@ -27,27 +27,35 @@ module "ecs" {
   source = "modules/ecs"
 
   environment          = "${var.environment}"
-  cluster              = "${var.environment}"
-  cloudwatch_prefix    = "${var.environment}"          #See ecs_instances module when to set this and when not!
+  cluster              = "${var.vpc_name}"
```

Comment:
The standard we have used in other examples is stack_name, would that be a better fit here?